src/Markup.php

```diff
@@ -27,6 +27,9 @@ public function __construct($content, $charset)
         $this->charset = $charset;
     }
 
+    /**
+     * @return string
```

Comment:
this type can be added directly (`__toString` is special in PHP and has an implicit `string` return type)